bot/seasons/evergreen/minesweeper.py

```diff
@@ -25,188 +46,188 @@ class CordConverter(commands.Converter):
         return x, y
 
 
+GameDict = typing.List[typing.List[typing.Union[str, int]]]
```

Comment:
This was fine being called `GameBoard`. My previous review was for `DictOfGames`, not this.